compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -6760,6 +6760,68 @@ TR::Node *laddSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    return node;
    }
 
+TR::Node *fpAddSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
```

Comment:
Can we give this a better name? I don't really like having both faddSimplifier and fpAddSimplifier . I know what the difference is, but the names don't have to be so similar which may confuse a less familiar reader. Maybe floatOrDoubleAddSimplifier or floatingPointAddSimplifier? Similar for the other examples. Do we have any consistent naming approach for this kind of thing that we could emulate here @andrewcraik @vijaysun-omr ?